src/test/feature/run-feature-test.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+
+if [ x$GPHOME == 'x' ]; then
+  echo "Please export GPHOME variable."
+  exit 0
+fi
```

Comment:
Actually for direct run, you need to source greenplum_path.sh at first, thus "source greenplum_path.sh" could be the same prerequisite for both direct run and parallel run.
We could check GP_HOME just to simply but not strictly judge whether users have sourced that file or not, not for us to source that file ourselves in the code.

Besides, another benefit is that you could make doc neat in README.MD :-) see the existing related doc below.

"Make sure HAWQ is running correctly. If not, init or start HAWQ at first.
Load environment configuration by running source $INSTALL_PREFIX/greenplum_path.sh.
Load hdfs configuration. For example, export HADOOP_HOME=/Users/wuhong/hadoop-2.7.2 && export PATH=${PATH}:${HADOOP_HOME}/bin. Since some test cases need hdfs and hadoop command, just ensure these commands work before running. Otherwise you will get failure."